### PR TITLE
feat(core.containers): HEALPixBeam container.

### PR DESCRIPTION
**Pull request by tristpinsm**
Used as an input to the driftscan beam transfer matrix calculation.

### CODE REVIEW

File: draco/core/containers.py [hunk 1]
```diff
@@ -781,7 +784,7 @@ class SiderealStream(VisContainer):
         The number of points to divide the RA axis up into.
     """
 
-    _axes = ("ra",)
+    _axes = ("ra", "freq", "stack", "input")
```
Comment (jrs65) [inline draco/core/containers.py:787]: Why did you change this? They're inherited from VisContainer.

File: draco/core/containers.py [hunk 2]
```diff
@@ -1064,6 +1067,73 @@ def phi(self):
...
+    def pol(self):
+        return self.index_map["pol"]
+
+    @property
+    def input(self):
+        return self.index_map["input"]
+
+    @property
+    def pix(self):
+        return self.index_map["pix"]
```
Comment (jrs65) [inline draco/core/containers.py:1130]: I probably wouldn't bother exposing this one as a property. It's not very useful to get an array of consecutive numbers. The only interesting thing is its length.

File: draco/core/containers.py [hunk 3]
```diff
@@ -1064,6 +1067,73 @@ def phi(self):
...
+            "distributed": True,
+            "distributed_axis": "freq",
+        },
+    }
+
+    def __init__(self, coords="unknown", ordering="unknown", *args, **kwargs):
+
+        super(HEALPixBeam, self).__init__(*args, **kwargs)
+        self.attrs["coords"] = coords
+        self.attrs["ordering"] = ordering
```
Comment (jrs65) [inline draco/core/containers.py:1098]: You should document (in the class docstring) what `coords` and `ordering` are for (listing possible values if that's reasonable).

**Review by jrs65 — CHANGES_REQUESTED** [review]

File: draco/core/containers.py (hunk 1 above)
Comment (tristpinsm) [reply draco/core/containers.py:787]: I don't know, this doesn't have anything to do with the PR, it must just have slipped in.

**Comment by jrs65** [comment]
@tristpinsm I'm trying to get through the massive back log of changes to merge. This looks good to me, but maybe you could quickly change it to be a subclass of the new HealpixContainer?

**Comment by tristpinsm** [comment]
ok, I did that, and it looks like the CI is happy now

**Comment by tristpinsm** [comment]
@jrs65 I can't merge until you approve

**Review by jrs65 — APPROVED** [review]
Approved. I just added a quick commit to use proper numpydoc format, squash it if you like.

**Comment by tristpinsm** [comment]
ok, thanks! black caught you leaving whitespace, so I'll fix that and squash everything into on commit.

**Pull request merged**